app/src/Toolbox.tsx

```diff
@@ -32,13 +30,19 @@ export function Toolbox({
         />
       </div>
       <div className="input-group mb-2">
-        <span className="input-group-text">Contract trait</span>
+        <span className="input-group-text">Interface</span>
         <Form.Select
           className="p-2"
           onChange={e => onContractTraitChanged(e.target.value)}
         >
           <option>Lottery</option>
           <option>Voting</option>
+          <option disabled>Auction</option>
+          <option disabled>MarketPlace</option>
+          <option disabled>FundMe</option>
+          <option disabled>Math</option>
+          <option disabled>PaymentSplit</option>
```

Comment:
Is this the full list of things or is this something provisional ?

Reply:
Is there a full list of things?

Reply:
I was thinking on this one we discussed in other channels. Not sure if want to put all of them.

Auction
Coin Flip
Lottery
MarketPlace
FundMe Contract
Raffle
Math (SafeMath + SafeDecimalMath)
Pause Contract - Allow pausing some transactional process
Payment splitter
reverse billing splitting
Voting facilities, create a smart contract that implements voting for different proposals, query the vote counting on each of them, etc ..

Reply:
This list is just temporary, I just summarized it to those easily understandable.